Userland/Utilities/comm.cpp

```diff
@@ -93,35 +103,41 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
     auto cmp = [&](String const& str1, String const& str2) {
         if (case_insensitive)
             return strcasecmp(str1.characters(), str2.characters());
-        else
-            return strcmp(str1.characters(), str2.characters());
-    };
-
-    auto process_remaining = [](String const& fmt, auto& file, int& count, bool print) {
-        while (file->can_read_line()) {
-            ++count;
-            auto line = file->read_line();
-            if (print)
-                outln(fmt, line);
-        }
+        return strcmp(str1.characters(), str2.characters());
     };
 
     bool read_file1 { true };
     bool read_file2 { true };
     int col1_count { 0 };
     int col2_count { 0 };
     int col3_count { 0 };
-    String file1_line, file2_line;
+    String file1_line;
+    String file2_line;
+    auto buffer = TRY(ByteBuffer::create_uninitialized(4096));
```

Comment:
Any reason you use 4096 instead of `PAGE_SIZE` like elsewhere?

Reply:
Good point, that's better!